file_example.c

```diff
@@ -360,33 +398,39 @@ static int file_handle_cmd(
 		return SAM_STAT_GOOD;
 	}
 	break;
+	case WRITE_VERIFY:
+		do_verify = true;
 	case WRITE_6:
 	case WRITE_10:
 	case WRITE_12:
 	case WRITE_16:
 	{
-		uint64_t offset = block_size * tcmu_get_lba(cdb);
-		int length = be16toh(*((uint16_t *)&cdb[7])) * block_size;
+		uint64_t cur_off = offset;
+		int length = tcmu_get_xfer_length(cdb) * block_size;
+		unsigned int to_copy;
+		int i = 0;
 
 		remaining = length;
 
-		while (remaining) {
-			unsigned int to_copy;
-
-			to_copy = (remaining > iovec->iov_len) ? iovec->iov_len : remaining;
+		tcmu_info("Enter write: %m\n");
```

Comment:
Hey.

Were the tcmu_info calls here and below supposed to be tcmu_dbg or deleted for the final merge?